Sources/Kumo/Services/Functions/Service+Upload.swift

```diff
@@ -10,62 +10,86 @@ public extension Service {
     ///   - endpoint: the path extension corresponding to the endpoint
     ///   - file: the URL of the file to upload
     ///   - key: the name of form part under which to embed the file's data
-    /// - Returns: an `Observable` which emits a single empty element upon success.
-    func upload(_ endpoint: String, parameters: [String: Any] = [:], file: URL, under key: String) -> Observable<Void> {
-        return Observable.create { [self] observer in
+    /// - Returns: an `AnyPublisher` which emits a single empty element upon success.
```

Comment:
> an `AnyPublisher` which publishes a single `Void` output upon success.

The documentation will need revised to use Combine language so that isn't not ambiguous or confusing. Probably can get away with doing this later, though.

Reply:
No particular rush to merge, I'll try to fix up the doc comments before merging. I probably left `emit` and `emits` in a lot of places, I tried to change the rest as I went but in my mind emit was still good (I agree with switching to publish now, just wasn't thinking about it before!)

Reply:
Looks like it was only two spots, done